packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -225,12 +225,16 @@ export enum ProcessStatus {
     COMPLETED = 'completed',
 }
 
+export interface State {
+    [index: string]: object;
```

Comment:
According to data on our test environment the type of the props can also be string, boolean, null (and maybe more)
![image](https://github.com/workfloworchestrator/orchestrator-ui/assets/20791917/ef3fe71c-bcfc-4f8f-9738-a98d28cfdc21)

Reply:
Yes; I wonder why the step state was defined in a way that it's incompatible with the graphql response in the first place...

Reply:
If you know a better solution please apply it in the MVP branch